sdk/typescript/src/builder/TransactionBlockData.ts

```diff
@@ -172,9 +168,11 @@ export class TransactionBlockDataBuilder {
   }
 
   build({
+    maxSizeBytes = Infinity,
     overrides,
     onlyTransactionKind,
   }: {
+    maxSizeBytes?: number;
```

Comment:
can this be `undefined` since we have a default above?

Reply:
This is just describing the type for the full options object, and doesn't know about the default (applied when destructuring) so this still needs to be optional.  `maxSizeBytes` is correctly typed as just `number` inside the function body.